docs/docs_requirements.txt

```diff
@@ -17,7 +17,6 @@ djangorestframework==3.0.3
 docker-py==0.6.0
 gunicorn==19.1.1
 paramiko==1.15.2
-psycopg2==2.5.4
```

Comment:
are there other requirements here which we could also remove, or will it need every other dependency to generate the controller autodocs?